src/follow-me/follow-me.js

```diff
@@ -124,6 +124,7 @@ function Follow( {
 	buttonSize,
 } ) {
 	const [ isOpen, setIsOpen ] = useState( false );
+	/* translators: %s: profile name */
 	const title = sprintf( __( 'Follow %s', 'activitypub' ), profile?.name );
```

Comment:
For the translator comment to be properly picked up by GlotPress, it would need to be right before the translatable string:

```suggestion
	const title = sprintf(
		/* translators: %s: profile name */
		__( 'Follow %s', 'activitypub' ),
		profile?.name
	);
```